### PR TITLE
feat(resources):  added support for identifier prefix for older esx versions.

### PR DESCRIPTION
**Pull request by iTexZoz**

### CODE REVIEW

**Comment by RockySouthpaw** [comment]
> > This is useful for other than people using older esx as there's variations of esx and peoples frameworks that don't do the whole string split. :)
> 
> I found the project interesting, I'm considering it to replace the gcphone on my server, however, having a very old version of ESX that has been modified all over the place by anyone, I thought this change would be useful for people other than me.

Thank you for the PR :). I myself don't split the string and normally just manually create a row in my db when i test the phone 😂 (since branch changing + modifing the code everytime isn't worth it.)

 I'll request code review from others, but this looks okay to me aside from `"useIdentifierPrefix"` being false by default since the current default is true.

File: config.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "RunRate": 10,
   "DestroyChance": 100,
   "DestroyPhoneReCheck": 3,
+  "useIdentifierPrefix": false,
```
Comment (itschip) [inline config.json:10]: This should go under database.

**Comment by RockySouthpaw** [comment]
@iTexZoz Can you apply the requested change so this can be merged?

**Review by RockySouthpaw — CHANGES_REQUESTED** [review]

**Comment by itschip** [comment]
@iTexZoz Are you able to resolve the current issue?

File: config.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (iTexZoz) [reply config.json]: Ho, sorry

**Comment by iTexZoz** [comment]
@itschip Sorry for the delay in answering, it's done!

**Review by TasoOneAsia — APPROVED** [review]

**Pull request merged**